<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nifo::ui::import_3dmodel_preset</class>
 <widget class="QWidget" name="nifo::ui::import_3dmodel_preset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <item>
       <widget class="QLabel" name="label_forward">
        <property name="text">
         <string>前向轴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>forward_axis</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="forward_axis">
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>+X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-Z</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
      <item>
       <widget class="QLabel" name="label_up">
        <property name="text">
         <string>向上坐标轴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>up_axis</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="up_axis">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>+X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-Z</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
      <item>
       <widget class="QLabel" name="label_scale">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>缩放</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="nifo::ui::floating_point_spinbox" name="scale" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="flip_uv">
      <property name="whatsThis">
       <string>沿y轴翻转UV坐标</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
      <property name="text">
       <string>UV翻转</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nifo::ui::floating_point_spinbox</class>
   <extends>QWidget</extends>
   <header>spinbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
